### ISSUE TITLE
all *.json files are interpreted as objects

### ISSUE THREAD

**Issue by spacek531** [OpenRCT2/OpenRCT2#16807]
### Operating System

Windows 10

### OpenRCT2 build

OpenRCT2, v0.3.5.1

### Describe the issue

All JSON files are loaded as if they are objects, causing issues when non-object JSON files exist in the object directory (e.g. sprites.json)

Potential fixes:

- verify json is dictionary not array
- verify objectType string exists

### Steps to reproduce

1. Have a sprites.json file in `object` dir
2. run game
3. game attempts to load json as object

**Comment by Gymnasiast** [comment]
Do you want to pick this up? Should be a simple fix. I think it should log to LOG_VERBOSE if it rejects a json file for these reasons.

**Comment by paul-harley** [comment]
Hi, I'm a student new to open source, would it be okay if I took this as my first issue?

**Comment by Gymnasiast** [comment]
Yes, go ahead. I think this should relatively straightforward. Please ask on Discord if you get stuck with anything.
